dist-material/release-docs/LICENSE

```diff
@@ -195,21 +195,104 @@ The text of each license is also included in licenses/LICENSE-[project].txt.
     https://npmjs.com/package/vis-timeline/v/7.5.1 7.5.1 (Apache-2.0 OR MIT)
     https://npmjs.com/package/vis-util/v/5.0.3 5.0.3 (Apache-2.0 OR MIT)
 
+========================================================================
```

Comment:
It seems you added a new zipkin-License, I think it is good.
Then the following new could be removed?

Reply:
We could add a line `Zipkin-lens dependences are attached in `zipkin-LICENSE` file. Apache 2.0`(Or something similar)